docs/admin/install.md

```diff
@@ -1,15 +1,30 @@
 # Installation Guide
 
-This document describes how to install a StashCache cache, a server that caches files locally rather than serving files to the StashCache federation. The installation utilizes XRootD and HTCondor for file storage and monitoring, respectively.
+This document describes how to install a StashCache service (Origin or Cache). The installation utilizes XRootD and HTCondor for file storage and monitoring, respectively.
```

Comment:
Can you describe the role of the Origin and Cache?  When would you need to install which?